CXL/ruleset.xml

```diff
@@ -171,4 +181,41 @@
         </properties>
     </rule>
 
+    <!-- Force phpDocs separation between different annotations and remove extra `*` -->
+    <!-- Require specific order of phpDoc annotations with empty newline between specific groups -->
+    <rule ref="SlevomatCodingStandard.Commenting.DocCommentSpacing">
+        <properties>
+            <property name="linesCountBetweenAnnotationsGroups" value="1"/>
+            <property name="linesCountBetweenDifferentAnnotationsTypes" value="1"/>
```

Comment:
@lkraav wrote on Slack "no empty lines". I'd change these to `0`

Reply:
> It makes it hard to read when multiple annotations are used, crowds the comment.

I agree in general. I'm just afraid that in most cases it can end up with something like this as we usually use 1-2 tags, so there's no need to separate them.

```
/**
 * @since 
 *
 * @param
 * 
 * @phpcsSuppress
 *
 * @throws
```